pom.xml

```diff
@@ -676,5 +676,16 @@
                 </plugins>
             </build>
         </profile>
+        <!-- For connecting to GCP CloudSQL Postgres instances. Needs 'boot' profile as well. -->
```

Comment:
You might consider putting the link you found here as well (the https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/main/docs/jdbc.md#postgres-1 one)

Reply:
Done